src/test/lsipii/User/StatusInfo.py

```diff
@@ -54,9 +54,9 @@ class StatusInfoRequest(CamelCaseModel):
 
 
 DEFINITION = DataProductDefinition(
-    summary="Data product for user's status information",
+    title="Data product for user's status information",
+    description="Retrieve user's status information",
```

Comment:
Based on how the other definitions were changed, I think the title and description should be swapped to be consistent with how it's done in the other definitions.